cluster/manager_delete.go

```diff
@@ -129,14 +129,14 @@ func deleteUserNamespaces(kubeConfig []byte, logger *logrus.Entry) error {
 				continue
 			default:
 				logger.Infof("namespace %q still %s", ns.Name, ns.Status)
-				left++
+				left = append(left, ns.Name)
 			}
 		}
-		if left > 0 {
-			return fmt.Errorf("%d namespaces remained after deletion", left)
+		if len(left) > 0 {
+			return fmt.Errorf("%d namespaces remained after deletion: %v", len(left), left)
```

Comment:
use `erros.Errorf` to capture stack info as well or `emperror.With(errors.New("namespaces remained after deletion"), "count", Len(left), "namespaces", left)`

Reply:
Thanks, added context/stack. I would keep the list in the message, so the user can see it without specifically handling this error somewhere in the callers.